Src/AutoFixture.NUnit3.UnitTest/PilotTest.cs

```diff
@@ -0,0 +1,14 @@
+﻿using NUnit.Framework;
+
+namespace Ploeh.AutoFixture.NUnit3.UnitTest
+{
+    [TestFixture]
+    public class PilotTest
+    {
+        [Test]
+        public void AssertTrue()
+        {
+            Assert.IsTrue(true);
```

Comment:
Nice comment! FWIW, if *I* ever win the lottery I'd contribute *only* to open-source (while also traveling around I guess)! 😀